protocol/testing/genesis.sh

```diff
@@ -77,9 +77,8 @@ function edit_genesis() {
 	dasel put -t string -f "$GENESIS" '.genesis_time' -v "$GENESIS_TIME"
```

Comment:
Update this file with the commit from the old version. 

Reason: the genesis file needs to be compatible with the old binary.